inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/db/RocksDbImp.java

```diff
@@ -160,23 +160,12 @@ public KeyValueEntity get(String key) {
 
     @Override
     public CommandEntity getCommand(String commandId) {
-        try {
-            byte[] bytes = db.get(columnHandlesMap.get(commandFamilyName), commandId.getBytes());
-            return bytes == null ? null : GSON.fromJson(new String(bytes), CommandEntity.class);
-        } catch (Exception e) {
-            throw new RuntimeException("get command value error", e);
-        }
+        return null;
     }
 
     @Override
     public CommandEntity putCommand(CommandEntity entity) {
-        requireNonNull(entity);
-        try {
-            db.put(columnHandlesMap.get(commandFamilyName), entity.getId().getBytes(), GSON.toJson(entity).getBytes());
-        } catch (Exception e) {
-            throw new RuntimeException("put value to rocks db error", e);
-        }
-        return entity;
+        return null;
```

Comment:
Sorry but why return null here?  This db store cmd from manager.

Reply:
the function will be delete, put an empty function here to make sure compilation passed